src/com/createsend/models/transactional/response/TransactionalClick.java

```diff
@@ -38,7 +38,7 @@
     @JsonProperty("Geolocation")
     private GeoLocation geoLocation;
 
-    @JsonProperty("URL")
+    @JsonProperty("Url")

```

Comment:
I'm curious about this change. Is the service sending payload containing `Url` now, instead of `URL`? If yes, why aren't the existing clients breaking?

Reply:
yeap, the sending payload has been using `Url` instead of `URL` for a while now. With the existing client, even if there is a value in the `Url`, the wrapper will return null because it cannot find any properties key of  `URL`